portlets/src/main/webapp/vue-app/realizations/components/Realizations.vue

```diff
@@ -202,13 +209,15 @@ export default {
         });
     },
     getRealizations() {
-      return this.$realizationsServices.getAllRealizations(this.fromDate, this.toDate, this.sortBy, this.sortDescending, this.offset, this.limit + 1)
+      this.returnType = 'json';
+      return this.$realizationsServices.getAllRealizations(this.fromDate, this.toDate,this.earnerId, this.sortBy, this.sortDescending, this.offset, this.limit + 1, this.returnType)
```

Comment:
no need to use a data for `this.returnType`. you can even not specify the `retrurnType` in REST call

```suggestion
      return this.$realizationsServices.getAllRealizations(this.fromDate, this.toDate,this.earnerId, this.sortBy, this.sortDescending, this.offset, this.limit + 1)
```